app/views/subscriptions_management/confirm_unsubscribe_all.html.erb

```diff
@@ -0,0 +1,36 @@
+<% content_for :title, 'Are you sure you want to unsubscribe from everything?' %>
+
+<% content_for :head do %>
+  <meta name="robots" content="noindex, nofollow">
+<% end %>
+
+<% content_for :head_css do %>
+  <%= stylesheet_link_tag 'application-without-elements' %>
+<% end %>
+
+<%= render 'govuk_publishing_components/components/back_link', {
+  href: @back_url,
+} %>
+
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <%= render 'govuk_component/title', title: 'Are you sure you want to unsubscribe from everything?' %>
+  </div>
+</div>
+
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <%= render 'govuk_component/govspeak', {
+      content: %{<p>You won’t get any more automated emails from GOV.UK.</p>},
+      rich_govspeak: true
```

Comment:
`rich_govspeak` only enables bold, doesn't look like there's any bold in the content.

Reply:
Ah I see! I'll change it to normal govspeak